tests/ui/E0599-write-macro/writeln-fmt-method-error.stderr

```diff
@@ -0,0 +1,13 @@
+error[E0599]: cannot write into `MyStruct`
+  --> $DIR/writeln-fmt-method-error.rs:23:5
+   |
+LL |     writeln!(s, "Hello, world!");
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+   = note: type does not implement the `write_fmt` method
+   = help: try adding `use std::fmt::Write;` or `use std::io::Write;` to bring the appropriate trait into scope
```

Comment:
`MyStruct ` doesn't implement either `Write`, so this doesn't help. The point of showing an error message that suggests importing traits is that we *know* the type implements the trait, but that the user cannot call its methods because the trait is not in scope. That's not the case here.